packages/engine/src/simulation/package/context/packages/api_requests/mod.rs

```diff
@@ -196,3 +190,32 @@ pub fn custom_message_handlers_from_properties(
         })
         .transpose()
 }
+
+async fn build_api_response_maps(
+    snapshot: &StateSnapshot,
+    handlers: &[String],
+) -> Result<Vec<ApiResponseMap>> {
+    tracing::warn!("BUILDIN API RESPONSE");
```

Comment:
What is this warning about? Looks like a println-style debug output.

Reply:
I sloppily left it in from some debugging. Darn. Thanks for spotting.
```suggestion
```